doc/releases/release-notes-3.6.rst

```diff
@@ -106,6 +106,8 @@ Boards & SoC Support
   * Added support for STM32L010xx SoC variants
   * Added support for STM32L081xx SoC variants
   * Added support for STM32U5A9xx SoC variants
+  * Added support for NXP S32K1 devices
+  * Added support for NXP IMX8ULP SoC
```

Comment:
Should we mention the support for the RT595 Fusion F1 DSP? That was added since last release